src/System.Diagnostics.Process/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

```diff
@@ -1,7 +1,6 @@
 advapi32.dll!OpenProcessToken
 kernel32.dll!GetProcessAffinityMask
-kernel32.dll!K32EnumProcessModules
 kernel32.dll!SetProcessAffinityMask
 kernel32.dll!SetThreadAffinityMask
-ntdll.dll!NtQueryInformationProcess
+kernel32.dll!QueryFullProcessImageName
```

Comment:
It's not in our snapshot of the WACK: https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.CodeAnalysis/PackageFiles/PinvokeAnalyzer_Win32UWPApis.txt  (I also checked the latest internal one and it's not there either). 

Try [GetModuleFileName](https://msdn.microsoft.com/en-us/library/windows/desktop/ms683197(v=vs.85).aspx) passing NULL for the first parameter. This is in the list.